methods_and_recursion/mad_libs.rb

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env ruby
+
+# Methods and Recursion
+# Homework #1
+# Pivotal Story 105170392
+
+# Welcoming statements - must hold off for cucumber tests
+# #puts
+# #puts 'Welcome to MadLibs'
+# #puts
+# #puts 'I am going to make a story with your words.'
+# #puts
+# #puts
+
+def ask_for(this_kind_of_word)
+  puts
```

Comment:
okay. now added the conventional puts '' for newlines \n
also rebased prior story acceptance, so Pull request is now ready for merge.